docs/ExpoManagedWorkflow.md

```diff
@@ -82,10 +86,12 @@ Instead of importing from `@appcues/react-native` directly, reference the `Appcu
 ```js
  import * as AppcuesWrapper from './AppcuesWrapper';
  await AppcuesWrapper.setup('APPCUES_ACCOUNT_ID', 'APPCUES_APPLICATION_ID');
- 
+
  AppcuesWrapper.identify('my-user-id');
- 
+
  AppcuesWrapper.screen('My Screen Name');
+
+ <AppcuesWrapper.WrappedAppcuesFrameView frameID="frame-id" />
```

Comment:
also I _think_ maybe if you do
```js
import { WrappedAppcuesFrameView } from './AppcuesWrapper';
```
then you can just use 
```js
<WrappedAppcuesFrameView frameID="frame-id" />
```
Though either way probably works fine
```

Reply:
Yes, that works too. I left it the way I did since the snippet already had the other import. I can add that as a little info callout.

Reply:
Added, and fixed the `!=` 4985f89 :+1:

Reply:
tested it all out locally and there was just one detail needed to make it work - added 
```js
import React from 'react';
```
in the wrapper script. That is needed for the `PlaceholderFrameView` to work when Appcues does not exist, presumably for the `View` usage (in addition to that import from react-native). Without that, the embed page errors out with "Can't find variable: React".